kbs/docs/kbs_attestation_protocol.md

```diff
@@ -155,23 +155,28 @@ evidence from the HW-TEE platform and organizes it into the following payload:
 
 ```json
 {
-  /*
-   * A JWK-formatted public key, generated by the KBC running in the HW-TEE.
-   * It is valid until the next time an attestation is required. Its hash must
-   * be included in the HW-TEE evidence and signed by the HW-TEE hardware.
-   */
-  /* Initdata plaintext */
   "init-data": {
-    "format": $initdata-format,
-    "body": $initdata-plaintext
+    "format": "json",
+    "body": "..."
   },
-  /* Runtime data plaintext */
-  "runtime-data": {},
-  /* The attestation evidence. Its format is specified by Attestation-Service. */
-  "tee-evidence": {}
+  "runtime-data": {
+    "nonce": "$nonce",
+    "tee-pubkey": {
+      "kty": "RSA",
+      "alg": "$key_algorithm",
+      "n": "$pubkey_modulus",
+      "e": "$pubkey_exponent"
```

Comment:
nit: maybe change this to the default used by kbs_protocol?

Reply:
Ok, let me do the change